tests/deprecated_api/test_remove_1-7.py

```diff
@@ -46,3 +48,33 @@ def test_v1_7_0_deprecated_model_size():
         match="LightningModule.model_size` property was deprecated in v1.5 and will be removed in v1.7"
     ):
         _ = model.model_size
+
+
+def test_v1_7_0_datamodule_transform_properties(tmpdir):
+    dm = MNISTDataModule()
+    with pytest.deprecated_call(match=r"DataModule property `train_transforms` was deprecated in v1.5"):
+        dm.train_transforms = "a"
+    with pytest.deprecated_call(match=r"DataModule property `val_transforms` was deprecated in v1.5"):
+        dm.val_transforms = "b"
+    with pytest.deprecated_call(match=r"DataModule property `test_transforms` was deprecated in v1.5"):
+        dm.test_transforms = "c"
+    with pytest.deprecated_call(match=r"DataModule property `train_transforms` was deprecated in v1.5"):
+        _ = LightningDataModule(train_transforms="a")
+    with pytest.deprecated_call(match=r"DataModule property `val_transforms` was deprecated in v1.5"):
+        _ = LightningDataModule(val_transforms="b")
+    with pytest.deprecated_call(match=r"DataModule property `test_transforms` was deprecated in v1.5"):
+        _ = LightningDataModule(test_transforms="c")
+
```

Comment:
Include dims here too

Reply:
@ananthsub how come? test the non-None `dims` construction in the `test_v1_7_0_datamodule_dims_property` method. is there a reason to test them all together?

Reply:
ah sorry I missed that below